python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/eos_cli_config_gen_documentation.schema.yml

```diff
@@ -20,3 +20,7 @@ keys:
         description: |
           Replace the input data using the `hide_passwords` filter in the Jinja2 templates by '<removed>' in the documentation if true.
         default: true
+      toc:
+        type: bool
+        default: true
+        description: Generate or skip the table of content(TOC) on fabric and device documentation.
```

Comment:
```suggestion
        description: Generate the table of content(TOC) on device documentation.
```

Reply:
updated and added the support for eos_designs